source/ETL/Data Pipeline/Jupyter Notebooks/ETL Log Reader SAS and SQL.ipynb

```diff
@@ -0,0 +1,4447 @@
+       "4           None       START        0.0              None             None   \n",
+       "...          ...         ...        ...               ...              ...   \n",
+       "1852        None       START        4.0              None             None   \n",
+       "1853        None       START        0.0              None             None   \n",
+       "1854        None    COMPLETE        0.0              None             None   \n",
+       "1855        None       start        1.0              None             None   \n",
+       "1856        None    COMPLETE        1.0              None             None   \n",
+       "\n",
+       "     Msg_Description2             create_dttm             update_dttm  \n",
+       "0                None 2024-01-07 15:53:40.970 2024-01-07 15:53:40.970  \n",
+       "1                None 2024-01-07 15:53:41.133 2024-01-07 15:53:41.133  \n",
+       "2                None 2024-01-07 15:53:41.140 2024-01-07 15:53:41.140  \n",
+       "3                None 2024-01-07 15:53:43.390 2024-01-07 15:53:43.390  \n",
+       "4                None 2024-01-07 15:53:43.407 2024-01-07 15:53:43.407  \n",
+       "...               ...                     ...                     ...  \n",
+       "1852             None 2024-01-08 13:52:53.917 2024-01-08 13:52:53.917  \n",
+       "1853             None 2024-01-08 14:13:05.363 2024-01-08 14:13:05.363  \n",
+       "1854             None 2024-01-08 14:13:05.573 2024-01-08 14:13:05.573  \n",
+       "1855             None 2024-01-08 14:13:05.773 2024-01-08 14:13:05.773  \n",
+       "1856             None 2024-01-08 14:13:05.777 2024-01-08 14:13:05.777  \n",
+       "\n",
+       "[1857 rows x 15 columns]"
+      ]
+     },
+     "execution_count": 64,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "sql_df_og"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 65,
+   "id": "b42a8d64",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql_df = sql_df_og"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 66,
+   "id": "2dc6a682",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Custom name updates to standardize logs. Common throughout all ETL runs. \n",
+    "update_d_labtest = ['D_LabTest', 'RDB.D_LabTest']\n",
+    "condition_d_labtest_1 = sql_df['Dataflow_Name'].isin(update_d_labtest)\n",
+    "condition_d_labtest_2 = sql_df['package_Name'].isin(update_d_labtest)\n",
+    "sql_df.loc[condition_d_labtest_1, 'Dataflow_Name'] = 'D_LABTEST'\n",
+    "sql_df.loc[condition_d_labtest_2, 'package_Name'] = 'RDB.D_LABTEST'\n",
+    "\n",
+    "update_Vaccination_Record = ['Vaccination_Record']\n",
+    "condition_Vaccination_Record = sql_df['Dataflow_Name'].isin(update_Vaccination_Record)\n",
+    "sql_df.loc[condition_Vaccination_Record, 'Dataflow_Name'] = 'VaccinationRecord'\n",
+    "\n",
+    "update_Lab100 = ['Lab100','RDB.D_LAB100']\n",
```

Comment:
Consider adding comments on which part of dimensions/datamarts is processed.

Reply:
Updated. Please take a look and let me know if it makes sense.